airbyte-integrations/connectors/source-mysql/build.gradle

```diff
@@ -9,7 +9,7 @@ application {
 airbyteBulkConnector {
     core = 'extract'
     toolkits = ['extract-jdbc', 'extract-cdc']
-    cdk = '0.249'
+    cdk = 'local'
```

Comment:
will need to replace this once https://github.com/airbytehq/airbyte/pull/50967 merges